install/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Installing</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Installing OpenShift Service Mesh</subtitle>
```

Comment:
Could be an attribute (last time I'll call this out).

Reply:
done